### DISCUSSION TITLE
Multiple filesystems

### DISCUSSION THREAD

**Discussion by jimmyspets** [seaweedfs/seaweedfs#6264] [Q&A] [answered]
I am considering switching from CephFS to SeaweedFS.
Today I have several different filesystems in one cluster with different properties.
One with only SSD for small files with a lot of random access and changes.
A second with big HDDs for large archive files that are written once and then never change.
How should that scenario be handled in SeeweedFS?
Should each filesystem have its own Master, Volume and Filer services?

**Comment by chrislusf** [comment, marked as answer]
you can configure different disk types for different directories.
